internal/cloudapi/v2/v2.go

```diff
@@ -363,6 +370,7 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 
 		irTarget = t
 	default:
```

Comment:
Is this a left-over? Should use the logger if we want this, no?

Reply:
oh no, sorry.  Leftover print debugging indeed.